packages/appsync-modelgen-plugin/src/visitors/appsync-visitor.ts

```diff
@@ -102,6 +102,14 @@ export interface RawAppSyncModelConfig extends RawConfig {
    * @descriptions optional boolean which adds the read-only timestamp fields or not
    */
   isTimestampFieldsAdded?: boolean;
+
+
+  /**
+   * @name directives
+   * @type boolean
+   * @description optional, defines if custom indexes defined by @key directive should be generated.
```

Comment:
It is better to clarify that this is only used in swift codegen in the description. Or if you implement parsing this config in swift visitor, you can move there this type definition as well